packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -45,8 +54,8 @@ export default class Atomic extends Command {
   }
 
   private async createProject() {
-    spawnProcess(appendCmdIfWindows`echo`, [
-      '...creating new Atomic project:',
+    return spawnProcess(appendCmdIfWindows`npx`, [
+      `${Atomic.cliPackage}@${getPackageVersion(Atomic.cliPackage)}`,
```

Comment:
I think this will fail in the E2E tests since the tests need to target a specific version (in Verdaccio).
You can solve that by adding a `--version` flag

```
public static flags = {
  version: flags.string({
    char: 'v',
    description: `The version of ${Atomic.cliPackage} to use.`,
    default: getPackageVersion(Atomic.cliPackage),
  }),
};
```

and then,

```
return spawnProcess(appendCmdIfWindows`npx`, [
      `${Atomic.cliPackage}@${flags.version}`,
```